doc/website/blog/title-tbd.md

```diff
@@ -0,0 +1,244 @@
+---
+title: "TBD"
+author: Jeremy Vesperman
+authorURL: "https://github.com/jeremyv2014"
+authorImageURL: "https://avatars.githubusercontent.com/u/9276415?v=4"
+---
+
+I have an embarrassing secret to share. I've been writing software for years, but, until recently, I 
```

Comment:
```suggestion
I have an embarrassing secret to shar: I've been writing software for years, but, until recently, I 
```

Reply:
@masseelch ? share is correct

Reply:
Yes, share is correct. I meant to swap the `.` with `:`, sorry.